ansible/roles/system_config/templates/nginx.j2

```diff
@@ -0,0 +1,11 @@
+server {
+
+    server_name DECIDIM_URL;
+
+    client_max_body_size 32M;
+
```

Comment:
Please set up 
- passenger_max_poll_size according to the following formula: (TOTAL_RAM * 0.75) / RAM_PER_PROCESS
- passenger_min_instances to the passenger_max_poll_size value

Reply:
@armandfardeau Did you test this Passenger config?